source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -0,0 +1,276 @@
+  ``IEnumerable`` interface
+
+Insert One Document
+~~~~~~~~~~~~~~~~~~~
+
+The following code shows how to use the asynchronous
+``InsertOneAsync()`` method or the synchronous ``InsertOne()`` method to
+insert one document.
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: insert-one-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         await _restaurantsCollection.InsertOneAsync(document);
+
+   .. tab:: Synchronous
+      :tabid: insert-one-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         _restaurantsCollection.InsertOne(document);
+
+Insert Multiple Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following code shows how to use the asynchronous
+``InsertManyAsync()`` method or the synchronous ``InsertMany()`` method to
+insert multiple documents.
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: insert-many-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         await _restaurantsCollection.InsertManyAsync(docs);
+
+   .. tab:: Synchronous
+      :tabid: insert-many-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         _restaurantsCollection.InsertMany(docs);
+
+.. tip::
+
+   Find runnable examples using these methods under :ref:`additional
+   information <additional-info>`.
+
+Parameters
+~~~~~~~~~~
+
+The ``InsertOne()`` method takes the document you seek to insert as a
+parameter. The ``InsertMany()`` method takes an ``IEnumerable``
+interface of documents, such as a list or array, as a parameter.
+
+The ``InsertOne()`` method optionally takes a ``InsertOneOptions`` type as an additional parameter,
+which represents options you can use to configure the insert operation.
+If you don't specify any ``InsertOneOptions`` properties, the driver does
+not customize the insert.
+
+The ``InsertOneOptions`` type allows you to configure options with the
+following properties:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``BypassDocumentValidation``
+     - | Gets or sets a value indicating whether to bypass document
+         validation. If ``true``, allows the write to opt-out of
+         :manual:`document level validation </core/schema-validation>`.
+
+   * - ``Comment``
+     - | Gets or sets the comment for the operation. See :manual:`the delete command
+         fields</reference/command/delete/#command-fields>`
+         for more information.
+
+The ``InsertMany()`` method optionally takes a ``InsertManyOptions``
+type as an additional parameter, which has the preceding
+``BypassDocumentValidation`` and ``Comment`` properties and the
+additional ``IsOrdered`` property:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``IsOrdered``
+     - | Gets or sets a value indicating whether the requests are
+         fulfilled in order. If ``true``, the driver sends documents to the
```

Comment:
Looking at that page, I think it may be a bit confusing to link to since it covers a lot more than the insert command. People clicking expecting to see more information specifically on the insert operation options may not get what they're looking for on that page